src/index.html

```diff
@@ -11,12 +11,111 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Consider using semantic HTML5 elements like `<nav>` for the navigation section to enhance the semantic structure of the document.